.github/workflows/test.yml

```
@@ -16,7 +16,7 @@ jobs:
     strategy:
       matrix:
         os: [Ubuntu]
-        python-version: ["3.8", "3.9", "3.10", "3.11"]
+        python-version: ["3.9", "3.10", "3.11", "3.12"]
```

3.8 is not yet EOL and is in security updates, so should probably do here:
```suggestion
        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12"]
```
Whenever we do drop 3.8 here it should be in conjunction with changing the min in `pyproject.toml` as we should test our oldest supported Python.